src/tribler-core/tribler_core/tests/test_start_core.py

```diff
@@ -1,29 +1,19 @@
 from unittest.mock import MagicMock, patch
 
-import pytest
-
+from tribler_core import start_core
 from tribler_core.start_core import start_tribler_core
-
-pytestmark = pytest.mark.asyncio
-
+from tribler_core.utilities.path_util import Path
 
 # pylint: disable=
 # fmt: off
 
-class MockedProcessChecker(MagicMock):
-    already_running = False
-
-
 @patch('tribler_core.load_logger_config', new=MagicMock())
-@patch('tribler_core.start_core.VersionHistory', new=MagicMock())
 @patch('tribler_core.start_core.set_process_priority', new=MagicMock())
 @patch('tribler_core.start_core.check_and_enable_code_tracing', new=MagicMock())
-@patch('tribler_core.start_core.core_session', new=MagicMock())
-@patch('tribler_core.start_core.ProcessChecker', new=MockedProcessChecker())
 @patch('asyncio.get_event_loop', new=MagicMock())
 @patch('tribler_core.start_core.TriblerConfig.load', new=MagicMock())
-@patch.object(MockedProcessChecker, 'remove_lock_file', create=True)
-async def test_start_tribler_core_no_exceptions(mocked_remove_lock_file):
+@patch.object(start_core, 'core_session')
```

Comment:
**Issue** it looks like an illegal way of using `patch.object`

You can use this instead:
```suggestion
@patch('tribler_core.start_core.core_session')
```